.pre-commit-config.yaml

```
@@ -88,3 +88,10 @@ repos:
     rev: v3.10.0-1
     hooks:
       - id: shfmt
+
+  - repo: https://github.com/ComPWA/taplo-pre-commit
+    rev: v0.9.3
+    hooks:
+      - id: taplo-format
+        # See options: https://taplo.tamasfe.dev/configuration/formatter-options.html
+        args: [--option, "reorder_arrays=true"]
```

With [fb0df95](https://github.com/pyvista/pyvista/pull/6781/commits/fb0df959950e6d2b6a2c1bbb02a24e99307f2ecd) I opted not to use `indent_string`, so the default indent of 2 is used. This is also the default for `toml-sort`. I think it's okay to use the default 2 spaces, but we can use 4 if preferred
```suggestion
        args: [--option, "reorder_arrays=true", --option, "indent_string=4"]
```

Have a look at the other [options](https://taplo.tamasfe.dev/configuration/formatter-options.html), are there any else we should include (can also add options in a separate PR)

---

The diff _is_ better with `"indent_string=    "` enabled if that's desired (i.e. fewer formatting changes). It's easier to see the sorting that way.

---

+1 using default.